projects/assets-library/src/icons/icon-library.module.ts

```diff
@@ -30,7 +30,7 @@ const iconsRootPath = 'assets/icons';
         { key: IconType.Close, url: `${iconsRootPath}/close.svg` },
         { key: IconType.CloseCircle, url: `${iconsRootPath}/close-circle.svg` },
         { key: IconType.CollapseAll, url: `${iconsRootPath}/collapse-all.svg` },
-        { key: IconType.Collapsed, url: `${iconsRootPath}/collapsed.svg` },
+        { key: IconType.Collapsed, url: `${iconsRootPath}/plus-square.svg` },
```

Comment:
Are the old SVGs still in use or can they be removed?

Reply:
will check that

Reply:
@aaron-steinfeld Removed unused icons